cns/client/client.go

```diff
@@ -1023,35 +1024,63 @@ func (c *Client) GetHomeAz(ctx context.Context) (*cns.GetHomeAzResponse, error)
 	return &getHomeAzResponse, nil
 }
 
+// GetEndpoint calls the EndpointHandlerAPI in CNS to retrieve the state of a given EndpointID
+func (c *Client) GetHNSEndpointID(ctx context.Context, endpointID string) ([]string, [][]net.IPNet, error) {
```

Comment:
can we add a brief comment on what this function returns

Reply:
funciton name is GetHNSEndpointID and it returns IPs too. lets consider renaming it  and  should we return as [] struct

Reply:
or a map[string][]net.IPNet

Reply:
Ended up removing the function since we can cover the previous version (with GetEndpoint+deleteEndpoint) in EndpointManger package without import cycle